ntoskrnl/lpc/reply.c

```diff
@@ -864,8 +864,8 @@ NtReplyWaitReplyPort(IN HANDLE PortHandle,
     PAGED_CODE();
     LPCTRACE(LPC_REPLY_DEBUG,
              "%s Attempted reply wait reply (Handle: %p, Message: %p)\n",
-	         PsGetCurrentProcess()->ImageFileName,
-	         PortHandle,
+             PsGetCurrentProcess()->ImageFileName,
+             PortHandle,
```

Comment:
Maybe squash with the previous commit? (the one modifying some other LPCTRACEs)

Reply:
Should look better now.